libass/ass.h

```diff
@@ -501,7 +517,7 @@ void ass_set_selective_style_override(ASS_Renderer *priv, ASS_Style *style);
  * \param priv renderer handle
  * \return success
  */
-int ass_fonts_update(ASS_Renderer *priv);
+ASS_DEPRECATED("No longer necessary") int ass_fonts_update(ASS_Renderer *priv);
```

Comment:
I don’t have a strong preference, but consider also the phrasing `"it does nothing"` (which I see in Apple’s Objective-C code).

Reply:
Changed this message and all messages now start with a lowercase letter.